index.html

```diff
@@ -249,5 +261,41 @@ <h3 class="connect-2">LET'S CONNECT</h3>
         <div></div>
       </div>
     </section>
+
+    <footer class="footer">
+      <div class="container-4">
+        <h1 class="first-title-3">Contact me</h1>
+        <p class="description-4">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+        <form class="contact-form" action="https://formspree.io/f/xyyokgln" method="get">
```

Comment:
- [ ] Kindly change your form method to `method=" post"` so that the user will be able to submit the form because currently your form is not being submitted successfully